Gateway/Command/Capture.php

```diff
@@ -197,6 +197,10 @@ public function execute(array $commandSubject)
 
         $this->paymentData->updateRemainingAmounts('capture', $amount, $paymentOrder);
 
+        $this->logger->debug('Saving sale transaction number to order grid!');
```

Comment:
It might be useful if the transaction number is logged as well, no?

```suggestion
        $this->logger->debug('Saving capture transaction number to order grid!');
```